plugins/source/aws/resources/services/lambda/functions_fetch.go

```diff
@@ -90,8 +91,8 @@ func resolvePolicyCodeSigningConfig(ctx context.Context, meta schema.ClientMeta,
 
 	// skip getting CodeSigningConfig since containerized lambda functions does not support this feature
 	// value can be nil if the caller doesn't have GetFunctionConfiguration permission and only has List*
-	lambdaType, ok := resource.Get("code_repository_type").(*string)
```

Comment:
I'm not that happy about `cqtypes` leaking into resolvers. How would a plugin author know which type to use in the casting?

I'd expect us to maintain the original SDK type (I understand this is a problem we had before, but `cqtypes` makes it worse I think).

Maybe we can change this to `r.Code.RepositoryType` (with `nil` checks)

Reply:
I think this is fine and we had the same behaviour before but actually even worth because you didn't really know what they type will be after it will be "resolved" but now it always the same per the definition of the column.